api/src/importers/coors/admin-sanction-utils.js

```diff
@@ -0,0 +1,155 @@
+        firstName: csvRow['first_name'] || '',
+        middleName: csvRow['middle_name'] || '',
+        lastName: csvRow['last_name'] || '',
+        dateOfBirth: csvRow['birth_date'] || null
+      };
+    }
+
+    adminSanction['location'] = csvRow['location_of_violation'] || '';
+
+    // Section 85 specific fields
+    if (csvRow['record_type_code'] === 'S85' && csvRow['business_reviewed_ind'] === 'Y') {
+      adminSanction['legislation'] = {
+        act: 'Wildlife Act',
+        regulation: '',
+        section: 85,
+        subSection: '',
+        paragraph: ''
+      };
+      adminSanction['legislationDescription'] = 'Angling, hunting and/or Limited Entry Hunting licence action for failure to pay fine';
+
+      adminSanction['penalties'] = [
+        {
+          type: 'Other',
+          penalty: {
+            type: 'Other',
+            value: null
+          },
+          description: 'Licences, LEH, Permits Cancelled'
+        }
+      ];
+      adminSanction['summary'] = this.buildSummary(csvRow, 85);
+    // Section 24 specific fields
+    } else if (csvRow['record_type_code'] === 'S24' && csvRow['business_reviewed_ind'] === 'Y') {
+      adminSanction['legislation'] = {
+        act: 'Wildlife Act',
+        regulation: '',
+        section: 24,
+        subSection: '',
+        paragraph: ''
+      };
+      adminSanction['legislationDescription'] = 'Angling, hunting, firearm and/or LEH licence action prompted by violations';
+
+      adminSanction['penalties'] = [
+        {
+          type: 'Other',
+          penalty: {
+            type: 'Other',
+            value: null
+          },
+          description: 'Suspension or cancellation of licence(s)'
+        }
+      ];
+      adminSanction['summary'] = this.buildSummary(csvRow, 24);
+    } else {
+      // row not reviewed by business, do not process
+      return null;
+    }
+
+    return adminSanction;
+  }
+
+  buildSummary(csvRow, sectionNum) {
+    let legislation;
+    if (!csvRow['regulation_description']) {
+      legislation = csvRow['act']
+    } else {
+      legislation = csvRow['regulation_description']
+    }
+
+    let enforcement = '';
+    if (csvRow['enforcement_licence_code'] === 'DDC') {
+      enforcement = "Director's decision to suspend licence";
+    } else if (csvRow['enforcement_licence_code' === 'AUT']) {
+      enforcement = 'Automatic licence suspension'
+    } else {
+      enforcement = 'Licence action'
+    }
+
+    const section = csvRow['section'] || '';
+    const sub_section = (csvRow['sub_section'] && ` & Sub-Section ${csvRow['sub_section']}`)|| '';
+    const paragraph = (csvRow['paragraph'] && ` & Paragraph ${csvRow['paragraph']}`)|| '';
+    const description = csvRow['violations_prompting_action'] || '';
+    let summary = '';
+    if (sectionNum === 85) {
+      summary = `Licence action resulting from an unpaid fine for an offense under the ${legislation} & Section ${section}${sub_section}${paragraph} - ${description}`;
```

Comment:
I don't think the `&` characters are supposed to be part of the final output.  For example, the output currently reads like "Angling and Scientific Collection Regulation & Section 17 & Sub-Section d".  It should probably read like "Angling and Scientific Collection Regulation Section 17 Sub-Section d"

Reply:
Updated to this format: Wildlife Act 26 (1) (C)